src/sdk/auth.ts

```diff
@@ -77,9 +81,12 @@ export const auth = {
         const query: Record<string, string> = {
           client_id: clientId,
           redirect_uri: redirectUrl,
-          scope: scopes.join(options.scopeSeparator ?? ' '),
+          scope: [...scopes, ...(options.defaultScopes ?? [])].join(
+            options.scopeSeparator ?? ' ',
+          ),
```

Comment:
I don't think defaultScopes should be required because it could be overly permissive?

We should use defaultScopes to serve as a template for new oauth apps?

Reply:
Ah it's not required? That's why we need to handle the fact that it can be null

Reply:
Gotcha. Sounds like we'll need to make sure the defaultScopes are actually requiredScopes right? Should we rename to `requiredScopes` then?

Additionally, it's possible that someone accidentally provides duplicate scopes. Should we do a unique?

Reply:
I don't think required is necessarily correct - they aren't necessarily required for the oauth to go through, they could just modify the behavior in some way that we want. Default is what it's called in other systems and I think in this context it makes the most sense. 

Can throw a unique around it 👍🏻

Reply:
> they could just modify the behavior in some way that we want
It doesn't seem like they can override it right? It seems like the only way they can modify is by pushing a code change which seems unideal.

What's the reason for not just having them define it when they create the oauth app? Do we have an app where we need this urgently?